src/docs/release/breaking-changes/platform-views-using-html-slots-web.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using HTML slots to render platform views in the web
+description: iframes in Flutter web used to reload, because of the way some DOM operations were made. A change in the way Flutter web apps render Platform Views makes them much more stable (preventing iframe reloads, and other problems with video tags or forms potentially losing their state).
+---
+
+## Summary
+
+Flutter now renders Platform Views using [slot elements][] inside of a single,
+app-wide [Shadow Root][]. Slot elements can be added/removed/moved around the
+Shadow DOM without affecting the underlying slotted content (which is rendered
+in a constant location)
+
+## Context
+
+The Flutter framework frequently tweaks its Render Tree to optimize the paint
+operations that are ultimately made per frame. In the web, these Render Tree
+changes often result in DOM operations.
+
+Flutter web used to render its Platform Views (`HtmlElementView` widgets)
+directly into its corresponding position of the DOM.
+
+Using certain DOM elements as the "target" of some DOM operations causes those
+elements to lose their internal state. In practice this means that `iframe` tags
+are going to reload, `video` players might restart, or a form being edited might
+lose the edits made to it.
+
+This change was made to:
+
+* make Platform Views behave in a stable way in Flutter web.
+* unify the way Platform Views were rendered in the web for both rendering
+backends (`html` and `canvaskit`).
+
+## Description of change
+
+A Flutter web app is now rendered inside a common Shadow Root in which slot
+elements are used to represent platform views. The actual content of each
+platform view is rendered *outside* of the Shadow Root, as a child of the Shadow
+Root host.
```

Comment:
I think we can gzip this sentence a bit: "...is rendered as a sibling of the shadow root"

Reply:
Or, since we're keeping `<flt-glass-pane>` we could just say "direct child of `<flt-glass-pane>`"